src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -76,7 +76,7 @@ def load_arguments(self, _):
         c.argument('no_ssh_key', options_list=['--no-ssh-key', '-x'])
         c.argument('pod_cidr')
         c.argument('service_cidr')
-        c.argument('vnet_subnet_id')
+        c.argument('vnet_subnet_id', type=str, validator=validate_vnet_subnet_id)
```

Comment:
Does it has a help message?

Reply:
yes it does, it keeps same as it was to be in _help.py

```
        - name: --vnet-subnet-id
          type: string
          short-summary: The ID of a subnet in an existing VNet into which to deploy the cluster.
```